docs/web-sdk/stopping-realtime-connection.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).